qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,17 +410,80 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, channel: int = None,
+                qubits: Union[Tuple[int], List[int]] = None) -> ControlChannel:
```

Comment:
How does this handle the scenario where multiple channels map to the same qubits?

Reply:
It will return all the channels associated with the same qubits since they can't be otherwise differentiated?

Reply:
List of control channels will be returned

Reply:
fixed in [c2649b4](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/c2649b40cfe0a9679661644414537f7621529ed2)